WORKSPACE

```diff
@@ -16,30 +16,72 @@ load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
 
 aspect_bazel_lib_dependencies()
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "sysroot",
+    build_file_content = """\
+filegroup(
+    name = "sysroot",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+""",
+    sha256 = "84656a6df544ecef62169cfe3ab6e41bb4346a62d3ba2a045dc5a0a2ecea94a3",
+    urls = ["https://commondatastorage.googleapis.com/chrome-linux-sysroot/toolchain/2202c161310ffde63729f29d27fe7bb24a0bc540/debian_stretch_amd64_sysroot.tar.xz"],
+)
+
 load("//toolchain:defs.bzl", "gcc_register_toolchain")
 
-flags = [
+common_flags = [
     "-Wall",
     "-Wextra",
     "-fdiagnostics-color=always",
 ]
 
+ldflags = common_flags + [
```

Comment:
nit: I think it's easier to reason about if you add `common_flags` to all or none of these other flag groups. currently this is the only one that doesn't get the `common_flags` added to it later when this symbol is referenced

Reply:
Same re cleaning up in the examples re-work I'm doing.